packages/sdk/src/actor/behaviors/penBehavior.ts

```diff
@@ -7,13 +7,13 @@ import { ToolBehavior } from '.';
 import { TransformLike } from '../..';
 
 
-interface DrawData {
+/* export */ interface DrawData {
```

Comment:
Any reason to still have these comments?

Reply:
Yes, because we haven't finished the 3D inking tool in Altspace yet, and it is not ready for the public.  We started on 3D inking and realized that 2D inking was more important at the moment, and that the button behavior could serve that purpose with some useful modifications.  I will be finishing up the 3D inking soon which is this PenBehavior.